backend/src/api/scans.ts

```diff
@@ -152,6 +152,13 @@ export const SCAN_SCHEMA: ScanSchema = {
     memory: '4096',
     description: 'Scrapes all webpages on a given domain, respecting robots.txt'
   },
+  hibp: {
+    type: 'fargate',
+    isPassive: true,
+    global: false,
+    description:
+      'Finds emails that have appeared in breaches related to a given domain'
```

Comment:
Since we're adding a new API key (HIBP_API_KEY), we need to set up the right variables on AWS / terraform infrastructure so that the deployed workers can also access HIBP_API_KEY. Check the instances of SHODAN_API_KEY and add a corresponding entry for HIBP_API_KEY:

https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/dev.env.example
https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/docs/src/documentation-pages/dev/own-instance.md
https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/infrastructure/stage.tfvars
https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/infrastructure/prod.tfvars
https://github.com/cisagov/crossfeed/blob/a70e552308996d7d396c101e331372122ab2fb0c/infrastructure/worker.tf

@DJensen94 not sure if you've worked with terraform before, so let me know if you'd like to maybe call / talk more about this so we can talk about how this should be set up.

Reply:
That would be great, I made the rest of the changes, just let me know when you have some time to jump on a call